internal/payloads/error_payload.go

```diff
@@ -63,6 +55,19 @@ func NewInvalidRequestError(ctx context.Context, err error) *ResponseError {
 	}
 }
 
+func NewMissingRequestParameterError(ctx context.Context, param string) *ResponseError {
+	msg := fmt.Sprintf("missing parameter: %s", param)
+	if logger := ctxval.Logger(ctx); logger != nil {
+		logger.Error().Msg(msg)
+	}
+	return &ResponseError{
+		HTTPStatusCode: 400,
+		Message:        msg,
+		RequestId:      ctxval.RequestId(ctx),
+		Context:        ctx,
+	}
+}
+
```

Comment:
The `ParamMissingError` looked weird, the purpose of this payloads package is to have payload errors, these are serializable JSON structures, not Go errors.